### PR TITLE
feat(config): add rekor endpoint env var

### PR DESCRIPTION
**Pull request by kahboom**
This PR adds the required environment variable in order for the `rekor-search-ui` to be able to issue requests to the Rekor server.

Original PR: https://github.com/securesign/sigstore-ocp/pull/204

Testing with the following container image for the rekor search UI: `quay.io/kahboom/rekor-search-ui@sha256:68a21610cd7de9c34466fed181fb3b26e527906e194f90657ab3c785b2151c85`

Instructions for testing: https://gist.github.com/kahboom/3294d64b5186b3c79b68bd78923ebec5

### CODE REVIEW

File: charts/trusted-artifact-signer/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 A Helm chart for deploying Sigstore scaffold chart that is opinionated for OpenShift
 
-![Version: 0.1.44](https://img.shields.io/badge/Version-0.1.44-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+![Version: 0.1.46](https://img.shields.io/badge/Version-0.1.46-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
```
Comment (kahboom) [inline charts/trusted-artifact-signer/README.md:6]: It brought in these changes when I ran the pre-commit hook described here: https://github.com/securesign/sigstore-ocp?tab=readme-ov-file#contributing

**Comment by kahboom** [comment]
I've added some instructions for testing the PR, hopefully it helps as I didn't previously share the container image for `rekor-search-ui` I had been using (sorry!): https://gist.github.com/kahboom/3294d64b5186b3c79b68bd78923ebec5#file-sigstore-ocp-md

**Comment by JasonPowr** [comment]
@kahboom The CI is currently failing with:
```
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "sigstore" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading scaffold from repo https://sigstore.github.io/helm-charts
Deleting outdated charts
Linting chart "trusted-artifact-signer => (version: \"0.1.46\", path: \"charts/trusted-artifact-signer\")"
Checking chart "trusted-artifact-signer => (version: \"0.1.46\", path: \"charts/trusted-artifact-signer\")" for a version bump...
Old chart version: 0.1.46
New chart version: 0.1.46
Error: failed linting charts: failed processing charts

------------------------------------------------------------------------------------------------------------------------
 ✖︎ trusted-artifact-signer => (version: "0.1.46", path: "charts/trusted-artifact-signer") > chart version not ok. Needs a version bump! 
------------------------------------------------------------------------------------------------------------------------
failed linting charts: failed processing charts
Error: Process completed with exit code 1.
```

Any time there is a change to the helm chart we need to bump the chart version by one: https://github.com/securesign/sigstore-ocp/blob/main/charts/trusted-artifact-signer/Chart.yaml#L36
so this should become: ```version: 0.1.47```

**Comment by JasonPowr** [comment]
@kahboom The changes themselves seem fine, works well :), just that small issue with the chart version

**Review by JasonPowr — APPROVED** [review]
/lgtm

**Pull request merged**